example/spec/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -238,6 +243,11 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
       });
     });
 
+    it('has expected ContentType values in s3', () => {
+      const checkContentType = headObjects.map((headObj) => headObj.expectedMime === headObj.ContentType);
+      checkContentType.forEach((eva) => expect(eva).toEqual(true));
```

Comment:
Good suggestion and good point. I chose to check at this point because move-granules has `movedFiles` readily accessible, but ultimately I think it's arbitrary.